lib/cassandra/cluster/schema/fetchers.rb

```diff
@@ -1278,7 +1278,13 @@ def create_column(column_data, types)
             name      = column_data['column_name']
             is_static = column_data['kind'].to_s.casecmp('STATIC').zero?
             order     = column_data['clustering_order'] == 'desc' ? :desc : :asc
-            type, is_frozen = @type_parser.parse(column_data['type'], types)
+            if column_data['type'][0] == "'"
+              # This is a custom column type.
+              type = Types.custom(column_data['type'].slice(1..-1).chomp("'"))
```

Comment:
better yet `str.slice(1, str.length - 2)` to avoid creating a new `Range` object